webviz_config/_build_webviz.py

```diff
@@ -22,22 +24,22 @@ def build_webviz(args: argparse.Namespace) -> None:
         raise ValueError(f"Theme `{args.theme}` is not installed.")
 
     if args.portable is None:
-        build_directory = tempfile.mkdtemp()
+        build_directory = pathlib.Path(tempfile.mkdtemp()).resolve().parent
     else:
-        build_directory = args.portable
+        build_directory = pathlib.Path(args.portable).resolve().parent
         os.makedirs(build_directory)
 
     shutil.copytree(
-        os.path.join(STATIC_FOLDER, "assets"), os.path.join(build_directory, "assets")
+        STATIC_FOLDER / "assets", build_directory / "assets")
     )
 
     for filename in ["README.md", "Dockerfile", ".dockerignore"]:
-        shutil.copy(os.path.join(STATIC_FOLDER, filename), build_directory)
+        shutil.copy(STATIC_FOLDER / filename, build_directory)
 
     for asset in installed_themes[args.theme].assets:
-        shutil.copy(asset, os.path.join(build_directory, "assets"))
+        shutil.copy(asset, build_directory / "assets")
 
-    with open(os.path.join(build_directory, "theme_settings.json"), "w") as filehandle:
+    with open(build_directory / "theme_settings.json", "w") as filehandle:
         filehandle.write(installed_themes[args.theme].to_json())
```

Comment:
Here we could perhaps utilize the [`path.write_text()`](https://docs.python.org/3/library/pathlib.html#pathlib.Path.write_text) method.

```suggestion
    (build_directory / "theme_settings.json").write_text(installed_themes[args.theme].to_json())
```